vimeo/VimeoNetworking
# Adding placeholder to OAuth process

#### Pull Request Checklist


#### Issue Summary

- OAuth process on web implemented three new parameters that can be passed as an url query in order to facilitate user login/join
1. Email -> A placeholder for email in the email field
2. Name -> A placeholder for name in the name field
3. Auth action -> Declare if user should see first the login screen or the join one

#### Implementation Summary

- We added three parameters in the method codeGrantAuthorizationURL in the AuthenticationController class. The method is now returning an url which include those parameters 

#### Reviewer Tips

Mainly the codeGrantAuthorizationURL method in the AuthenticationController class was modified.
Another helper method was added

#### How to Test

1. Unit test was added and should pass
2. Add parameters to codeGrantAuthorizationURL method and check if, when Safari is opened, the placeholders are in place
